cmd/minikube/cmd/console.go

```diff
@@ -47,14 +52,24 @@ var consoleCmd = &cobra.Command{
 		}
 		if consoleURLMode {
 			fmt.Fprintln(os.Stdout, url)
+		} else if machineReadAble{
```

Comment:
Testing would get much easier if you take as much as possible out of 'func(cmd *cobra.Command, args []string) '. Split this up into multiple functions. For example split this into two methods. One to get the URL using libmachine client and _cluster.GetConsoleURL_. This part won't be testable unless you mock. Second method is _displayConsoleInformation(url string)_. This just gets the URL and is concerned about putting this to screen in the right format. You can test the right format gets printed now without mocking anything. You can test _displayConsoleInformation_. Just capture standard out and make sure the result is correct.